rct229/rulesets/ashrae9012019/section12/section12rule3.py

```diff
@@ -11,46 +33,204 @@ class Section12Rule3(RuleDefinitionListIndexedBase):
+                    index_rmd=PROPOSED,
+                    list_path="$.miscellaneous_equipment[*]",
                 )
 
-            def get_calc_vals(self, context, data=None):
+            def create_data(self, context, data):
+                space_p = context.PROPOSED
+
                 return {
-                    "user_space_id": context.USER["id"],
+                    "space_type_p": getattr_(space_p, "spaces", "lighting_space_type")
                 }
 
-            def rule_check(self, context, calc_vals, data):
-                return calc_vals["user_space_id"] in data["proposed_space_ids"]
+            class MiscEquipRule(RuleDefinitionBase):
+                def __init__(self):
+                    super(
+                        Section12Rule3.RuleSetModelDescriptionRule.SpaceRule.MiscEquipRule,
+                        self,
+                    ).__init__(
+                        rmds_used=produce_ruleset_model_description(
+                            USER=False, BASELINE_0=True, PROPOSED=True
+                        ),
+                        required_fields={
+                            "$": [
+                                "has_automatic_control",
+                                "multiplier_schedule",
+                            ]
+                        },
+                        manual_check_required_msg="Credit for automatic receptacle controls was expected, but baseline and proposed miscellaneous equipment schedules are identical.",
+                    )
+
+                def get_calc_vals(self, context, data=None):
+                    misc_equip_b = context.BASELINE_0
+                    misc_equip_p = context.PROPOSED
+
+                    is_leap_year = data["is_leap_year"]
+                    space_type_p = data["space_type_p"]
+                    schedule_b = data["schedule_b"]
+                    schedule_p = data["schedule_p"]
+
+                    if (
+                        space_type_p not in EXPECTED_RECEPTACLE_CONTROL_SPACE_TYPES
+                        and misc_equip_p["has_automatic_control"]
+                    ):
+                        expected_receptacle_power_credit = 0.1 * getattr_(
+                            misc_equip_p,
+                            "miscellaneous_equipment",
+                            "automatic_controlled_percentage",
+                        )
+
+                        hourly_multiplier_schedule_b = misc_equip_b[
+                            "multiplier_schedule"
+                        ]
+                        hourly_multiplier_schedule_p = misc_equip_p[
+                            "multiplier_schedule"
+                        ]
+
+                        expected_hourly_values = [
+                            hour_value * (1 - expected_receptacle_power_credit)
+                            for hour_value in schedule_b[hourly_multiplier_schedule_b]
+                        ]
```

Comment:
@JacksonJ-KC @KarenWGard 
I do have a question here.
For a case when the `hour_value` in propose model is 1, how does it match to baseline case?
My suggestion would be flip the comparison in the RDS
```python
expected_hourly_values_in_b = [
                            min(1, hour_value * (1 + expected_receptacle_power_credit))
                            for hour_value in schedule_p[hourly_multiplier_schedule_p]
                        ]
credit_comparison_data = compare_schedules(
                            expected_hourly_values_in_p,
                            schedule_b[hourly_multiplier_schedule_b],
                            mask_schedule,
                            is_leap_year,
                        )["total_hours_matched"]
```

Reply:
I am not following the question or the proposed change to the RDS. These are the 3 cases.
1) comparison of **proposed** hourly values and **expected proposed** hourly values. If all hours match - pass.
2) comparison of **proposed** hourly values and **baseline** hourly values. If all hours match - undetermined
3) fail

The expected proposed hourly values are the baseline values * (1 - expected_receptacle_power_credit)

Reply:
Typically, user develop proposed model first then baseline model.
If the multiplier value is 1.0 in an hour on a day in proposed model, then the baseline values would be proposed values * (1 + expected_receptacle_power_credit), this sets the multiplier in baseline over the 1.0 threshold.

What I suggest is to check in a reverse order, the baseline value matches to `min(1.0,  proposed value * (1+expected_receptacle_power_credit))`

Reply:
If the proposed model used 1 as the multiplier value when credit was expected then they should receive a fail outcome

Reply:
![image](https://github.com/user-attachments/assets/5cb99142-b492-4d33-b0a0-f223d7c6b486)

Reply:
This aligns with Standard 90.1 where the expected proposed schedule is based on the baseline * ( 1 - RPC )

Reply:
If you prefer to check that the baseline matches expected values then your revised equation `expected baseline values = proposed values / (1 - expected_receptacle_power_credit)` would be the correct one. If an expected value is ever greater than 1 then that means credit was forgotten and should be either undetermined if all values match baseline, otherwise fail

Reply:
Note this can still lead to expected values that are greater than 1 and you **should not** replace the expected values greater than 1 with 1. If an expected value is greater than 1 then either Case 2 is met or Fail.

Reply:
I think this would be a problem since it relates to software capabilities and circular reference in the standard.
I would suggest adding an undetermined for scenario where when resetting multiplier > 1.0 to 1.0 matches to proposed and add a message such as "Hours matched only when resetting greater than 1.0 hourly multipliers to 1.0"
This case is different from CASE 2 since a good chunk of hours would be matched to `credit_comparison_data` but only the portion that are over 1.0 falls the matching.

Reply:
Note to discuss at the next RDS meeting

Reply:
The next RDS meeting won't happen in two weeks so I open an issue #1510 for the discussion later.

Reply:
I'll address this issue when the issue is resolved.